prometheus-server-metrics/src/main/scala/org/http4s/server/prometheus/PrometheusExportService.scala

```diff
@@ -0,0 +1,66 @@
+package org.http4s.server.prometheus
+
+import java.io.StringWriter
+
+import cats.effect._
+import cats.implicits._
+import io.prometheus.client.CollectorRegistry
+import io.prometheus.client.exporter.common.TextFormat
+import io.prometheus.client.hotspot._
+import org.http4s._
+import org.http4s.dsl.Http4sDsl
+
+class PrometheusExportService[F[_]: Sync] private (
+    s: HttpService[F],
+    cr: CollectorRegistry
```

Comment:
If these are vals, you don't need the defs.

This could be a final class.

Reply:
I agree, but unless you want pain on the way forward, seems like that should be done after we have gotten this through.